python_ta/z3/z3_parser.py

```diff
@@ -15,37 +15,26 @@ class Z3ParseException(Exception):
     pass
 
 
-class ExprWrapper:
+class Z3Parser:
     """
-    Wrapper class to convert an astroid expression node into a z3 expression.
+    Convert an astroid expression node into a z3 expression.
```

Comment:
The description of a class should fundamentally be a _noun_, not a verb. It is okay to remove "Wrapper" though, so "Class that converts" would be okay.